addon/components/admin-input.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-var set = Ember.set;
-var get = Ember.get;
-var computed = Ember.computed;
+const set = Ember.set;
+const get = Ember.get;
+const computed = Ember.computed;
```

Comment:
When run test in chrome. But i forget run in terminal, sorry.
I try tomorrow again.